packages/commerce-sdk-react/src/hooks/createUseQuery.ts

```diff
@@ -0,0 +1,154 @@
+/*
```

Comment:
General questions:

1. Why did you decide to put this into it's own file? I ask because this feels like a utility function, and we have a util file in this folder. Also, the parent folder is called "hooks" so ideally it only contains hooks, but I know that we have strayed from that path, but this would mean we are staying even more.
2. Unit Tests?

Reply:
Thanks for the review!

1. I don't feel strongly and happy to move it to a util files. I just thought that this is the core implementation of this library and it is very important to make sure it's visible and easy to find as oppose to buried in the utils file that usually contains a bunch of functions that are unrelated to each other and are usually less important stuff.
2. Added tests

Reply:
1. I wont dye on that hill. 2. Thanks for adding the tests.